src/Management/src/TracingBase/TracingBaseServiceCollectionExtensions.cs

```diff
@@ -86,7 +86,10 @@ public static IServiceCollection AddDistributedTracing(this IServiceCollection s
 
                 if (traceOpts.PropagationType.Equals("B3", StringComparison.InvariantCultureIgnoreCase))
                 {
+                    // TODO: Investigate alternatives and remove suppression.
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
FYI @hananiel, looks like we need to look at B3 headers and see where things are going